shared/src/main/scala/effekt/symbols/symbols.scala

```diff
@@ -304,6 +336,7 @@ package object symbols {
   }
 
   lazy val Pure = new Effects(Nil)
+
```

Comment:
This seams to be an useless blank line. What's your policy about this? Okay or Nay?

Reply:
No policy, but Effectful is super important and shouldn't be too close to another definition.